diff -uNr openocd-0.10.0/src/target/target.c openocd-0.10.0_patched/src/target/target.c
--- openocd-0.10.0/src/target/target.c	2016-12-25 15:12:56.000000000 +0100
+++ openocd-0.10.0_patched/src/target/target.c	2019-05-23 15:37:04.000000000 +0200
@@ -104,6 +104,7 @@
 extern struct target_type or1k_target;
 extern struct target_type quark_x10xx_target;
 extern struct target_type quark_d20xx_target;
+extern struct target_type hinetiol_target;
 
 static struct target_type *target_types[] = {
 	&arm7tdmi_target,
@@ -135,6 +136,7 @@
 	&or1k_target,
 	&quark_x10xx_target,
 	&quark_d20xx_target,
+	&hinetiol_target,
 	NULL,
 };
 
