diff -uNr openocd-0.10.0/src/server/server.c openocd-0.10.0_patched/src/server/server.c
--- openocd-0.10.0/src/server/server.c	2016-12-25 15:12:56.000000000 +0100
+++ openocd-0.10.0_patched/src/server/server.c	2019-05-23 15:37:04.000000000 +0200
@@ -108,7 +108,7 @@
 
 #ifdef _WIN32
 		/* we are using stdin/out so ignore ctrl-c under windoze */
-		SetConsoleCtrlHandler(NULL, TRUE);
+		//SetConsoleCtrlHandler(NULL, TRUE);
 #endif
 
 		/* do not check for new connections again on stdin */
@@ -549,21 +549,21 @@
 	WSADATA wsaData;
 
 	wVersionRequested = MAKEWORD(2, 2);
-
+	// negotite socket version. required before any sockat action on windows see: https://docs.microsoft.com/en-us/windows/win32/api/winsock/nf-winsock-wsastartup
 	if (WSAStartup(wVersionRequested, &wsaData) != 0) {
 		LOG_ERROR("Failed to Open Winsock");
 		exit(-1);
 	}
 
 	/* register ctrl-c handler */
-	SetConsoleCtrlHandler(ControlHandler, TRUE);
-
-	signal(SIGBREAK, sig_handler);
-#endif
-	signal(SIGINT, sig_handler);
-	signal(SIGTERM, sig_handler);
-	signal(SIGABRT, sig_handler);
-
+	//SetConsoleCtrlHandler(ControlHandler, TRUE);
+
+	//signal(SIGBREAK, sig_handler);
+#endif
+	//signal(SIGINT, sig_handler);
+	//signal(SIGTERM, sig_handler);
+	//signal(SIGABRT, sig_handler);
+	LOG_OUTPUT("No signal handler catched. Ctrl+C should work. Expects to terminate programm immidiatly!");
 	return ERROR_OK;
 }
 
@@ -583,7 +583,7 @@
 
 #ifdef _WIN32
 	WSACleanup();
-	SetConsoleCtrlHandler(ControlHandler, FALSE);
+	//SetConsoleCtrlHandler(ControlHandler, FALSE);
 
 	return ERROR_OK;
 #endif
